recipe/meta.yaml

```diff
@@ -39,7 +42,7 @@ requirements:
     - json-c   # [not win]
     - kealib
     - lerc
-    - libcurl 7.88.1
+    - libcurl 8.1.1
```

Comment:
@cbouss using 8 is problematic because so far we've ben using 7. Most packages depend on 7. https://metayaml-conda.fly.dev/metayaml/reverse_dependencies?name=libcurl

Reply:
7 gives >=7,<8, which is translated to >=7,<9 via repodata hotfix. 
This does gives us a wider range but relies on a hotfix... We could rebuild the last libcurl 7 with <9.

Reply:
I'll leave it pinned to 8.

Reply:
Yep, works for me! Thanks for checking.